src/UnitTests/UnitTests.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="log4net" Version="2.0.12" />
```

Comment:
Please remove `log4net` since it is already a transitive dependency because of project-reference.